<launch>
  <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find planning_exec_interface)/config/logger.conf"/>

  <node pkg="planning_exec_interface" name="planning_execution"
        type="planning_execution" output="screen">
    <rosparam>
      names_group:
        - "arm"
        #- "positioner"
      num_dof_groups:
        - 6
        #- 2
    </rosparam>
  </node>
</launch>
